db/db_with_timestamp_basic_test.cc

```diff
@@ -590,8 +590,130 @@ TEST_F(DBBasicTestWithTimestamp, CompactDeletionWithTimestampMarkerToBottom) {
 
 class DataVisibilityTest : public DBBasicTestWithTimestampBase {
  public:
-  DataVisibilityTest() : DBBasicTestWithTimestampBase("data_visibility_test") {}
+  DataVisibilityTest() : DBBasicTestWithTimestampBase("data_visibility_test") {
+    // Initialize test data
+    for (int i = 0; i < kTestDataSize; i++) {
+      test_data_[i].key = "key" + ToString(i);
+      test_data_[i].value = "value" + ToString(i);
+      test_data_[i].timestamp = Timestamp(i, 0);
+      test_data_[i].ts = i;
+      test_data_[i].seq_num = kMaxSequenceNumber;
+    }
+  }
+
+ protected:
+  struct TestData {
+    std::string key;
+    std::string value;
+    int ts;
+    std::string timestamp;
+    SequenceNumber seq_num;
+  };
+
+  constexpr static int kTestDataSize = 3;
+  TestData test_data_[kTestDataSize];
+
+  void PutTestData(int index, ColumnFamilyHandle* cfh = nullptr) {
```

Comment:
Nit: I am doubtful about the benefit of default argument here. Explicitly specifying `cfh` is clearer.

Reply:
if `cfh` is not specified, then it writes to default cf, I think it's consistent with our other APIs. Otherwise, we have to change all
```
PutTestData(1);
```
to:
```
PutTestData(1, nullptr);
```